lib/install/validate/network.go

```diff
@@ -501,24 +501,26 @@ func (v *Validator) dpgMorefHelper(op trace.Operation, ref string) (string, erro
 	return ref, nil
 }
 
-func (v *Validator) dpgHelper(op trace.Operation, path string) (types.ManagedObjectReference, error) {
+func (v *Validator) dpgAndOpaqueHelper(op trace.Operation, path string) (object.NetworkReference, error) {
```

Comment:
Should this be `*object.NetworkReference` instead? That's what we're type checking in the function.
```suggestion
func (v *Validator) dpgAndOpaqueHelper(op trace.Operation, path string) (*object.NetworkReference, error) {
```

Reply:
Will fix it!